java/vector/src/main/java/org/apache/arrow/vector/complex/ListVector.java

```diff
@@ -297,6 +298,9 @@ public int getBufferSize() {
 
   @Override
   public Field getField() {
+    if (getDataVector() == ZeroVector.INSTANCE) {
```

Comment:
I know that the `vector` is initialized to `ZeroVector.INSTANCE` right now, but would this be a little better as  `if (getDataVector() instanceof ZeroVector)`?

Reply:
What was the previous field name?

Reply:
I believe it was the `ZeroVector.name` which is "[DEFAULT]"